.gitignore

```diff
@@ -1,6 +1,2 @@
-node_modules
-
-# vuepress build output
-docs/.vuepress/dist
```

Comment:
Why did you remove this? I think it makes sense to keep it because we don't want to publish the vuepress build output.

Reply:
because those files are just need it to build the file, they are not part of the source code. If you commit the `node_modules` you would be commiting actually all those dependencies. The practice is just to keep the reference to them in the `package.json`.

Reply:
Yes, I know. That's why node_modules was already present in .gitignore before. But your commit removed the line `docs/.vuepress/dist` from .gitignore. I think it should be kept, same as `yarn.lock`.

Reply:
Ok this is now corrected.